js/script.js

```diff
@@ -1,5 +1,7 @@
 var a = 2,
 	h = 5,
-	triangleArea = a*h/2;
+	triangleArea = a*h/2,
```

Comment:
instead last `,` should be a `;`, because after triangleArea you end your variable declarations ;)